src/surfing_penguin/__init__.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask
```

Comment:
Headers should be added as well.
See https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files